### PR TITLE
Skull Sceptre Item Charges

### PR DESCRIPTION
**Pull request by FreckledDad**
This allows for the skull sceptre to display its charges.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -35,20 +35,22 @@
 	<description>Open source RuneScape client</description>
 	<url>http://runelite.net</url>
 
-	<properties>
```
Comment (Adam-) [inline pom.xml:38]: remove all modifications to this file.

File: runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemChargeConfig.java [hunk 2]
```diff
@@ -47,6 +47,7 @@
 	String KEY_EXPLORERS_RING = "explorerRing";
 	String KEY_RING_OF_FORGING = "ringOfForging";
 	String KEY_BLOOD_ESSENCE = "bloodEssence";
+	String KEY_SKULL_SCEPTRE_I = "skullSceptre(i)";
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemChargeConfig.java:50]: I don't know if () is safe here, it is probably best to not include them

File: runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemChargeType.java [hunk 3]
```diff
@@ -52,7 +52,8 @@
 	RING_OF_FORGING(ItemChargeConfig::showRingOfForgingCount),
 	POTION(ItemChargeConfig::showPotionDoseCount),
 	GUTHIX_REST(ItemChargeConfig::showGuthixRestDoses),
-	BLOOD_ESSENCE(ItemChargeConfig::showBloodEssenceCharges);
+	BLOOD_ESSENCE(ItemChargeConfig::showBloodEssenceCharges),
+	INVOKE(ItemChargeConfig::showSkullSceptreCount);
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemChargeType.java:56]: `INVOKE`? what is that?

**Review by Adam- — CHANGES_REQUESTED** [review]

**Comment by makingcomment** [comment]
Why only skull sceptre (i) and not the regular (non-imbued) one?

**Comment by FreckledDad** [comment]
Well the non imbued version gets deleted on last teleport use so not sure
why you'd ever be using that.

File: runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemChargeType.java (hunk 3 above)
Comment (FreckledDad) [reply runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemChargeType.java:56]: INVOKE is the teleport option

File: pom.xml (hunk 1 above)
Comment (FreckledDad) [reply pom.xml:38]: Reverted

File: runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemChargeConfig.java (hunk 2 above)
Comment (FreckledDad) [reply runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemChargeConfig.java]: Removed ()

File: pom.xml (hunk 1 above)
Comment (Adam-) [reply pom.xml:38]: I'd recommend looking at the diff between what you are pring and master - https://github.com/runelite/runelite/pull/14661/files - you can see that the changes haven't been reverted

**Pull request closed without merging**